Synthesis failed for nodejs-logging

Hello! Autosynth couldn't regenerate nodejs-logging. :broken_heart:

Please investigate and fix this issue within 5 business days.  While it remains broken,
this library cannot be updated with changes to the nodejs-logging API, and the library grows
stale.

See https://github.com/googleapis/synthtool/blob/master/autosynth/TroubleShooting.md
for trouble shooting tips.

Here's the output from running `synth.py`:

```
.md
2021-03-30 04:21:35,196 synthtool [INFO] > Replaced 'required_envvars[^\\)]*\\)' in .trampolinerc.
INFO:synthtool:Replaced 'required_envvars[^\\)]*\\)' in .trampolinerc.
2021-03-30 04:21:35,196 synthtool [INFO] > Replaced 'pass_down_envvars\\+\\=\\(' in .trampolinerc.
INFO:synthtool:Replaced 'pass_down_envvars\\+\\=\\(' in .trampolinerc.
2021-03-30 04:21:35,196 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2021-03-30 04:21:35,196 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN deprecated cheerio-select-tmp@0.1.1: Use cheerio-select instead

> protobufjs@6.10.2 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging/node_modules/protobufjs
> node scripts/postinstall


> @google-cloud/logging@9.1.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.1.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src

installing tmp@^0.2.0
installing uglify-js@^3.7.7
installing espree@^7.0.0
installing escodegen@^2.0.0
installing estraverse@^5.1.0

> @google-cloud/logging@9.1.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.1.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.3.1 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 712 packages from 829 contributors and audited 725 packages in 34.2s

83 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2021-03-30 04:22:10,039 synthtool [DEBUG] > Running prelint...
DEBUG:synthtool:Running prelint...

> @google-cloud/logging@9.1.0 prelint /home/kbuilder/.cache/synthtool/nodejs-logging
> cd samples; npm link ../; npm install


> @google-cloud/logging@9.1.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.1.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src


> @google-cloud/logging@9.1.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.1.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

audited 725 packages in 19.713s

83 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

/home/kbuilder/.nvm/versions/node/v14.16.0/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
/home/kbuilder/.cache/synthtool/nodejs-logging/samples/node_modules/@google-cloud/logging -> /home/kbuilder/.nvm/versions/node/v14.16.0/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.3.1 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 250 packages from 170 contributors and audited 350 packages in 7.484s

91 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2021-03-30 04:22:38,878 synthtool [DEBUG] > Running fix...
DEBUG:synthtool:Running fix...

> @google-cloud/logging@9.1.0 fix /home/kbuilder/.cache/synthtool/nodejs-logging
> gts fix

version: 14

/home/kbuilder/.cache/synthtool/nodejs-logging/env-tests-logging/deployable/nodejs/app.js
   1:27  error  "@google-cloud/logging" is not found  node/no-missing-require
  12:36  error  'context' is defined but never used   no-unused-vars

✖ 2 problems (2 errors, 0 warnings)

Error: Command failed with exit code 1: node ./node_modules/eslint/bin/eslint --fix **/*.ts **/*.js **/*.tsx **/*.jsx --no-error-on-unmatched-pattern
    at makeError (/home/kbuilder/.cache/synthtool/nodejs-logging/node_modules/gts/node_modules/execa/lib/error.js:59:11)
    at handlePromise (/home/kbuilder/.cache/synthtool/nodejs-logging/node_modules/gts/node_modules/execa/index.js:114:26)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async run (/home/kbuilder/.cache/synthtool/nodejs-logging/node_modules/gts/build/src/cli.js:120:17) {
  shortMessage: 'Command failed with exit code 1: node ./node_modules/eslint/bin/eslint --fix **/*.ts **/*.js **/*.tsx **/*.jsx --no-error-on-unmatched-pattern',
  command: 'node ./node_modules/eslint/bin/eslint --fix **/*.ts **/*.js **/*.tsx **/*.jsx --no-error-on-unmatched-pattern',
  exitCode: 1,
  signal: undefined,
  signalDescription: undefined,
  stdout: undefined,
  stderr: undefined,
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging@9.1.0 fix: `gts fix`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging@9.1.0 fix script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2021-03-30T11_23_21_553Z-debug.log
2021-03-30 04:23:21,562 synthtool [ERROR] > Failed executing npm run fix:

None
ERROR:synthtool:Failed executing npm run fix:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging/synth.py", line 53, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 249, in postprocess_gapic_library
    fix(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 183, in fix
    shell.run(["npm", "run", "fix"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'run', 'fix']' returned non-zero exit status 1.
2021-03-30 04:23:21,603 autosynth [ERROR] > Synthesis failed
2021-03-30 04:23:21,604 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 54b8ff3 chore: add in updated compiled protos (#1012)
2021-03-30 04:23:21,624 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-03-30 04:23:21,650 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing build/
Removing node_modules/
Removing package-lock.json
Removing samples/node_modules/
Removing samples/package-lock.json
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/54d497ed-bd76-4445-92b7-317396052bf9/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging;failed=false).

Autosynth is still having trouble generating nodejs-logging. :sob:

Please investigate and fix this issue within 5 business days.  While it remains broken,
this library cannot be updated with changes to the nodejs-logging API, and the library grows
stale.

See https://github.com/googleapis/synthtool/blob/master/autosynth/TroubleShooting.md
for trouble shooting tips.

Here's the output from running `synth.py`:

```
] > Replaced 'required_envvars[^\\)]*\\)' in .trampolinerc.
INFO:synthtool:Replaced 'required_envvars[^\\)]*\\)' in .trampolinerc.
2021-03-31 04:22:22,279 synthtool [INFO] > Replaced 'pass_down_envvars\\+\\=\\(' in .trampolinerc.
INFO:synthtool:Replaced 'pass_down_envvars\\+\\=\\(' in .trampolinerc.
2021-03-31 04:22:22,279 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2021-03-31 04:22:22,279 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN deprecated cheerio-select-tmp@0.1.1: Use cheerio-select instead

> protobufjs@6.10.2 postinstall /home/kbuilder/.cache/synthtool/nodejs-logging/node_modules/protobufjs
> node scripts/postinstall


> @google-cloud/logging@9.1.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.1.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src

installing tmp@^0.2.0
installing uglify-js@^3.7.7
installing espree@^7.0.0
installing escodegen@^2.0.0
installing estraverse@^5.1.0

> @google-cloud/logging@9.1.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.1.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.3.1 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 712 packages from 830 contributors and audited 725 packages in 35.989s

83 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2021-03-31 04:22:58,949 synthtool [DEBUG] > Running prelint...
DEBUG:synthtool:Running prelint...

> @google-cloud/logging@9.1.0 prelint /home/kbuilder/.cache/synthtool/nodejs-logging
> cd samples; npm link ../; npm install


> @google-cloud/logging@9.1.0 prepare /home/kbuilder/.cache/synthtool/nodejs-logging
> npm run compile-protos && npm run compile


> @google-cloud/logging@9.1.0 compile-protos /home/kbuilder/.cache/synthtool/nodejs-logging
> compileProtos src


> @google-cloud/logging@9.1.0 precompile /home/kbuilder/.cache/synthtool/nodejs-logging
> gts clean

version: 14
Removing build ...

> @google-cloud/logging@9.1.0 compile /home/kbuilder/.cache/synthtool/nodejs-logging
> tsc -p . && cp -r proto* build/

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

audited 725 packages in 20.337s

83 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

/home/kbuilder/.nvm/versions/node/v14.16.0/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
/home/kbuilder/.cache/synthtool/nodejs-logging/samples/node_modules/@google-cloud/logging -> /home/kbuilder/.nvm/versions/node/v14.16.0/lib/node_modules/@google-cloud/logging -> /home/kbuilder/.cache/synthtool/nodejs-logging
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.3.1 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 250 packages from 170 contributors and audited 350 packages in 7.647s

91 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

2021-03-31 04:23:28,655 synthtool [DEBUG] > Running fix...
DEBUG:synthtool:Running fix...

> @google-cloud/logging@9.1.0 fix /home/kbuilder/.cache/synthtool/nodejs-logging
> gts fix

version: 14

/home/kbuilder/.cache/synthtool/nodejs-logging/env-tests-logging/deployable/nodejs/app.js
   1:27  error  "@google-cloud/logging" is not found  node/no-missing-require
  12:36  error  'context' is defined but never used   no-unused-vars

✖ 2 problems (2 errors, 0 warnings)

Error: Command failed with exit code 1: node ./node_modules/eslint/bin/eslint --fix **/*.ts **/*.js **/*.tsx **/*.jsx --no-error-on-unmatched-pattern
    at makeError (/home/kbuilder/.cache/synthtool/nodejs-logging/node_modules/gts/node_modules/execa/lib/error.js:59:11)
    at handlePromise (/home/kbuilder/.cache/synthtool/nodejs-logging/node_modules/gts/node_modules/execa/index.js:114:26)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async run (/home/kbuilder/.cache/synthtool/nodejs-logging/node_modules/gts/build/src/cli.js:120:17) {
  shortMessage: 'Command failed with exit code 1: node ./node_modules/eslint/bin/eslint --fix **/*.ts **/*.js **/*.tsx **/*.jsx --no-error-on-unmatched-pattern',
  command: 'node ./node_modules/eslint/bin/eslint --fix **/*.ts **/*.js **/*.tsx **/*.jsx --no-error-on-unmatched-pattern',
  exitCode: 1,
  signal: undefined,
  signalDescription: undefined,
  stdout: undefined,
  stderr: undefined,
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/logging@9.1.0 fix: `gts fix`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/logging@9.1.0 fix script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2021-03-31T11_24_12_899Z-debug.log
2021-03-31 04:24:12,908 synthtool [ERROR] > Failed executing npm run fix:

None
ERROR:synthtool:Failed executing npm run fix:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-logging/synth.py", line 53, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 249, in postprocess_gapic_library
    fix(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 183, in fix
    shell.run(["npm", "run", "fix"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'run', 'fix']' returned non-zero exit status 1.
2021-03-31 04:24:12,958 autosynth [ERROR] > Synthesis failed
2021-03-31 04:24:12,958 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 0157ae1 fix: Insufficient return values for `GetEntriesResponse` and `GetLogsResponse` (#1011)
2021-03-31 04:24:12,979 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-03-31 04:24:13,006 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing build/
Removing node_modules/
Removing package-lock.json
Removing samples/node_modules/
Removing samples/package-lock.json
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/28e4a3a0-a971-4a94-8fe4-7387dacc1c26/targets/github%2Fsynthtool;config=default/tests;query=nodejs-logging;failed=false).

--- a/.eslintignore
+++ b/.eslintignore
@@ -4,4 +4,4 @@ test/fixtures
 build/
 docs/
 protos/
-env-tests-logging/
+**/env-tests-logging

--- a/.kokoro/environment.sh
+++ b/.kokoro/environment.sh
@@ -38,8 +38,8 @@ fi
 npm install
 npm run compile
 
-# Add the test module as a submodule to the repo root
-git submodule add https://github.com/googleapis/env-tests-logging
+# make sure submodule is up to date
+git submodule update --init --recursive
 cd "env-tests-logging/"
 
 # Disable buffering, so that the logs stream through.

--- a/.prettierignore
+++ b/.prettierignore
@@ -4,4 +4,4 @@ test/fixtures
 build/
 docs/
 protos/
-env-tests-logging/
+**/env-tests-logging

--- a/.prettierignore
+++ b/.prettierignore
@@ -4,3 +4,4 @@ test/fixtures
 build/
 docs/
 protos/
+env-tests-logging/

--- a/synth.py
+++ b/synth.py
@@ -35,6 +35,7 @@ common_templates = gcp.CommonTemplates()
 templates = common_templates.node_library(source_location='build/src')
 s.copy(templates, excludes=[
     ".eslintignore",
+    ".prettierignore",
     "CONTRIBUTING.md"
 ])
 

--- a/synth.py
+++ b/synth.py
@@ -28,7 +28,7 @@ gapic = gcp.GAPICBazel()
 version='v2'
 # tasks has two product names, and a poorly named artman yaml
 v2_library = gapic.node_library("logging", version, proto_path=f'google/logging/{version}')
-s.copy(v2_library, excludes=["src/index.ts", "README.md", "package.json", "system-test/fixtures/sample/src/index.js", "system-test/fixtures/sample/src/index.ts"])
+s.copy(v2_library, excludes=[".eslintignore", ".prettierignore", "src/index.ts", "README.md", "package.json", "system-test/fixtures/sample/src/index.js", "system-test/fixtures/sample/src/index.ts"])
 
 # Copy in templated files
 common_templates = gcp.CommonTemplates()

